cmd/kubeadm/app/master/selfhosted_volumes_test.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package master
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
```

Comment:
I'm ok with what is here, b/c you'd need more integration testing to vet the other f(n)'s but we should follow on with another PR once sig-auth has audited this one.